src/main/java/com/aws/greengrass/componentmanager/ComponentStore.java

```diff
@@ -226,15 +230,42 @@ ComponentRecipe getPackageRecipe(@NonNull ComponentIdentifier pkgId) throws Pack
      * @param compId component identifier
      * @throws PackageLoadingException if deletion of the component failed
      */
-    void deleteComponent(@NonNull ComponentIdentifier compId) throws PackageLoadingException {
+    void deleteComponent(@NonNull ComponentIdentifier compId,
+                         @NonNull ArtifactDownloaderFactory artifactDownloaderFactory)
+            throws PackageLoadingException, InvalidArtifactUriException {
         logger.atDebug("delete-component-start").kv("componentIdentifier", compId).log();
         IOException exception = null;
+        // issues #1111
```

Comment:
Do we want to remove this comment?

Reply:
No strong opinion from me. Maybe it doesn't hurt to keep a ref to the issue number?